episodes/episodes.Rmd

```diff
@@ -1073,13 +1074,120 @@ pie(
+
+{sandpaper} can automatically generate glossary links in lessons from links to Glosario terms in lesson content markdown.
+
+### Configuration
+
+Firstly, to enable this behaviour, the `glosario` key needs to be added to a lesson's `config.yaml`.
+
+For automatic retrieval of the latest `glossary.yml` from Glosario's GitHub repository, simply set:
+
+```yaml
+glosario: true
+```
+
+For manual retrieval from a local (e.g. if you are working on your lesson offline) or remote location (e.g. providing a specific Glosario version), supply a string relative/absolute path or URL to the config:
+
+```yaml
+# load a glossary from a local file using an absolute path
+glosario: "/path/to/your/glossary.yml"
+
+# load a glossary from a local file using path relative to the lesson
+glosario: "../glossary.yml"
+
+# load a glossary from a remote location
+glosario: "https://foo.com/glossary.yml"
+```
+
+### Lesson content
+
+For example, the [datacarpentry/spreadsheet-ecology-lesson](https://github.com/datacarpentry/spreadsheet-ecology-lesson/blob/main/episodes/01-format-data.md) contains the following markdown:
+
+```
+The most common mistake made is treating spreadsheet programs like lab notebooks, that is, relying on context,
+notes in the margin, spatial layout of data and fields to convey information. As humans, we can (usually) interpret
+these things, but computers don't view information the same way, and unless we explain to the computer what
+every single thing means (and that can be hard!), it will not be able to see how our data fits together.
+
+Using the power of computers, we can manage and analyze data in much more effective and faster ways, but to
+use that power, we have to set up our data for the computer to be able to understand it (and computers are very literal).
+```
+
+A lesson maintainer/contributor can add placeholders into the main content of lessons that sandpaper will subsequently process.
+
+Sandpaper understands links to Glosario terms in two ways, either a template-style term, or a hard link to the term page, i.e.:
+
+- `{{ glosario.<term> }}`
+- `[<term>](https://glosario.carpentries.org/en/#<term>)`
+
+:::::::::::::: callout
+
+## Automatic language selection
+
+When using the template-style term, sandpaper will automatically generate the URL link using the language specified in the `config.yaml` `lang` setting.
+
+When using the markdown link style, sandpaper will automatically replace the `en/` in the link with the language specified in the `config.yaml` `lang` setting.
+
+For example, if the `config.yaml` language is set to `lang: de`:
+
+`[data_structure](https://glosario.carpentries.org/en/#data_structure)`
+
+will be replaced with:
+
+`[data_structure](https://glosario.carpentries.org/de/#data_structure)`
+
+If the term is not available in that language, a warning will be printed when the lesson builds, and it will fall back to English.
```

Comment:
```suggestion
If the term is not available in that language, a warning will be printed when the lesson builds, and it will default back to English.
```